spaces/serializers.py

```diff
@@ -0,0 +1,47 @@
+from rest_framework import serializers
+
+from .models import Address, Place
+
+
+class AddressSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Address
+        fields = ('street_name', 'civic', 'floor', 'door', 'city', 'postal_code', 'region', 'nation')
+
+
+class PlaceSerializer(serializers.HyperlinkedModelSerializer):
+    address = AddressSerializer()
+
+    class Meta:
+        model = Place
+        fields = ('url', 'name', 'owner', 'description', 'address')
+
+    def update(self, instance, validated_data):
```

Comment:
According to http://www.django-rest-framework.org/api-guide/serializers/#modelserializer

The ModelSerializer class is the same as a regular Serializer class, except that:
* [...]
* **It includes simple default implementations of .create() and .update().**

You don't seem to be doing anything that the ModelSerializer would not already do for you.